src/app/components/search/SearchResults.js

```diff
@@ -381,27 +405,47 @@ function SearchResultsComponent({
       );
     }
   } else {
-    content = (
-      <SearchResultsTable
-        projectMedias={projectMedias}
+    content = resultType === 'factCheck' ? (
+      <SearchResultsCards
         team={team}
-        selectedIds={filteredSelectedProjectMediaIds}
-        sortParams={sortParams}
-        onChangeSelectedIds={handleChangeSelectedIds}
-        onChangeSortParams={handleChangeSortParams}
-        buildProjectMediaUrl={buildProjectMediaUrl}
-        resultType={resultType}
-        count={count}
+        projectMedias={projectMedias}
       />
+    ) : (
+      <div style={{ overflow: 'auto' }}>
+        { projectMedias.map((item) => {
+          // It appears that list_columns_values gotta be a json string in the optimistic object
+          const list_columns_values = typeof item.list_columns_values === 'string' ? JSON.parse(item.list_columns_values) : item.list_columns_values;
```

Comment:
Looks like `list_columns_values` is being deprecated per https://github.com/meedan/check-api/pull/1771/files